src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -351,6 +377,8 @@ export default class OrgPerson extends Mixins(CommonMixin) {
    * Called when user clicks Done button.
    */
   private validateOrgPersonForm (): void {
+    this.applyValidation()
```

Comment:
At first glance, this seems like it should be awaited.

It probably works anyway because the `validate()` calls below just handle the new rules. I'm not entirely sure.

I think it's good form to await an async function in all cases.

Reply:
kk, will update